From fd8ef554e17a165f3c437cda958cdb58ff813258 Mon Sep 17 00:00:00 2001
From: Rajesh Gugulothu <gugulothu.rajesh@xilinx.com>
Date: Tue, 27 Oct 2020 10:07:06 -0700
Subject: [PATCH 1732/1852] v4l: xilinx: Fix compatible string

commit 4c4408c0489be2c2fcca8ebb74d67225d5063a6d from
https://github.com/Xilinx/linux-xlnx.git

Fix the device tree compatible string based on the device tree generator

Signed-off-by: Rajesh Gugulothu <gugulothu.rajesh@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-dprxss.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/xilinx/xilinx-dprxss.c b/drivers/media/platform/xilinx/xilinx-dprxss.c
index e907f2ca5f96..dde0c2fcda26 100644
--- a/drivers/media/platform/xilinx/xilinx-dprxss.c
+++ b/drivers/media/platform/xilinx/xilinx-dprxss.c
@@ -1296,7 +1296,7 @@ static int xdprxss_remove(struct platform_device *pdev)
 }
 
 static const struct of_device_id xdprxss_of_id_table[] = {
-	{ .compatible = "xlnx,v-dp-rxss-2.1", },
+	{ .compatible = "xlnx,v-dp-rxss-3.0", },
 	{ /* end of table */ }
 };
 MODULE_DEVICE_TABLE(of, xdprxss_of_id_table);
-- 
2.31.1

